_gcode/G034-zsaa.md

```diff
@@ -38,4 +38,4 @@ example:
 
 ---
 
-Align multiple Z stepper motors using a bed probe by probing one position per stepper. See [`M422`](/docs/gcode/M422.html) for Z-Stepper automatic alignment parameter selection.
+Align multiple Z stepper motors using a bed probe by probing one position per stepper. See [`M422`](/docs/gcode/M422.html) for Z-Stepper automatic alignment parameter selection. `G28` is done beforehand and is implicit. Also see `HOME_AFTER_G34` in `Configuration_adv.h`
```

Comment:
`G28` is only done if needed/axis positions are unknown, so this isn’t exactly correct.

Reply:
I now see the comment in G28: `Homing is required before [G29](https://marlinfw.org/docs/gcode/G029.html), [M48](https://marlinfw.org/docs/gcode/M048.html), and some other procedures.`

IMHO it would be worth it to figure out which procedures auto-home if required pr prefer to fail first, maybe standardize this